docs/userguide.md

```diff
@@ -1,71 +1,106 @@
 
-## Overview
-* short one
+## Install
+---
 
+=== "Installation & Updates"
 
-## Walkthrough
+	* Choose the programs you want to install or upgrade.
+		* For programs not currently installed, this action will install them.
+		* For programs already installed, this action will update them to the latest version.
+	* Click the `Install/Upgrade Selected` button to start the installation or upgrade process.
 
-### Program
+=== "Upgrade All"
 
-#### Installation & Updates
-* To install programs select the programs you wish to install or update like the picture below.
-![Program Install](assets/ProgramInstall.png)
-* Once you have selected the programs you wish to install click the select Install/Upgrade Selected button as seen below.
-![Program Install Button](assets/ProgramInstallButton.png)
+    * Simply press the `Upgrade All` button.
+	* This will upgrade all applicable programs that are installed without the need for individual selection.
 
-#### Upgrade All
-* Press the button to upgrade all installed programs that are supported by WinGet, there is no selection needed.
+=== "Uninstall"
 
-#### Uninstall
-* To uninstall programs select the programs you wish to uninstall like the picture below.
-![Program Uninstall](assets/ProgramInstall.png)
-* Once you have selected the programs you wish to uninstall click the select Uninstall Selected button as seen below.
-![Program Uninstall Button](assets/ProgramUninstall.png)
+    * Select the programs you wish to uninstall.
+	* Click the `Uninstall Selected` button to remove the selected programs.
 
-#### Get Installed
-* Checks for installed programs that are supported by WinGet and selects them in the Utility.
+=== "Get Installed"
 
-#### Clear Selection
-* Clears ur current selection so no program is checked.
+    * Click the `Get Installed` button.
+	* This will scan for and selects all installed programs in WinUtil that WinGet supports.
 
-### Tweaks
+=== "Clear Selection"
 
-#### Tweaks Addition
-* To enable tweaks on your system select Tweaks at the top next to Install.
-* Then you can select what tweaks you want adding to your system. We do have some presets you can select from at the top you can see this in the picture below.
-![Tweaks Screen](assets/TweaksScreen.PNG)
-* After you have chosen your tweaks click the Run Tweaks button at the bottom of the screen.
+    * Click the `Clear Selection` button.
+	* This will deselect all checked programs.
 
-#### Tweaks Removal
-* To disable tweaks on your system select Tweaks at the top next to Install.
-* Then you can select what tweaks you want removing from your system.
-* After you have chosen your tweaks you want to remove click the Undo Selected Tweaks button at the bottom of the screen.
+![Install Image](assets/Install-Tab-Dark.png#only-dark)
+![Install Image](assets/Install-Tab-Light.png#only-light)
+* If you have trouble finding an application press `ctrl + f` and search the name of it, applications will filter depending on your input.
```

Comment:
```suggestion
!!! tip

   If you have trouble finding an application press `ctrl + f` and search the name of it, applications will filter depending on your input.
```
This will make this statement clear-er to the reader, as currently it's doesn't show it as a list, _**see screenshot below**_.

![image](https://github.com/user-attachments/assets/d3ee8966-0420-467d-b0c3-0fb337d87c74)